CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Contributing to Project-Euler-Library-Code
+ - [Pull Request Guidelines](#pull-request-guidelines)
+ - [Rebasing / Squashing Commits](#rebasing-squashing-commits)
+ - [Commit Message Format](#commit-message-format)
+ 
+## Pull Request Guidelines
+Pull requests should be created with a short description in the title.
+
+The body of the pull request can contain a larger description of the changes and 
+screenshots, if necessary. The associated branch can have one or more commits until it has at least one thumbs up, until we have more contributors.
+```
+<Short Description>
+<BLANK LINE>
+<Longer Description>
+(<Screenshots>)
+```
+
+After one or more core team members have given it a **Thumbs Up**:
+1. [Rebase](#rebasing-squashing-commits) the branch locally into a single commit following the [commit message format](#commit-message-format).
+2. Force Push the local, rebased branch back to Git.
+3. If the Merge Request still displays the old commits, then close and re-open the request.
+ 
+## Rebasing / Squashing Commits
+1. Fetch the latest changes by going to VCS->Git->Fetch
+2. Check out the latest version of master to your local branches.
+3. Go to VCS->Git->Rebase.
```

Comment:
If that's too complicated, I think we might want to use a simpler method of merging in pull requests for a standard pull request.